client/app/scripts/superdesk-dictionaries/views/settings.html

```diff
@@ -1,38 +1,33 @@
 <div sd-settings-view>
 
 	<div class="split-content" sd-dictionary-config>
-		
+
 		<div class="header">
 			<h2 translate>Dictionary management</h2>
 			<button class="btn btn-info pull-right" ng-click="createDictionary()">
 		        <i class="icon-plus-sign icon-white"></i> {{ 'Add New Dictionary' | translate }}
 		    </button>
 		</div>
-		
+
 	    <div class="content dictionary-settings">
 	    	<div sd-shadow>
 	    		<ul class="pills-list">
 					<li ng-repeat="dictionary in dictionaries._items">
```

Comment:
I know, but I mentioned it nevertheless so that it doesn't get forgotten. It's a trivial change anyway.